lib/manageiq/api/client/mixins/acts_as_ar_query_mixin.rb

```diff
@@ -0,0 +1,14 @@
+module ActsAsArQueryMixin
+  extend ActiveSupport::Concern
+
+  def ar_query
```

Comment:
This should probably be a private method.  Also, it feels strange...we don't have to do this in ManageIQ where ActsAsArQuery is used.